pkg/apis/kubebind/v1alpha1/apiserviceexport_types.go

```diff
@@ -90,8 +90,25 @@ type APIServiceExportSpec struct {
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="informerScope is immutable"
 	InformerScope Scope `json:"informerScope"`
+
+	// ClusterScopedIsolation specifies how cluster scoped service objects are isolated between multiple consumers on the provider side.
+	// It can be "Prefixed", "Namespaced", or "None".
+	ClusterScopedIsolation Isolation `json:"clusterScopedIsolation"`
 }
 
+type Isolation string
```

Comment:
use kubebuilder enum marker to define which values are valid in OpenAPI.

Reply:
Done in the latest commit.